README.md

```diff
@@ -64,7 +64,7 @@ Run `.\build-windows.ps1` in a new powershell terminal to build the source with
 
 ### Code Modification
```

Comment:
@ethanmoffat Why is this in the README to begin with - is there any reason not to just add the include unconditionally?

Reply:
Agree, this change should be made in the code. Making code changes to a repo you clone should be unnecessary.

If there is an incompatibility with this `#include` statement between different versions of MSVC, add `#ifdef` guards that check the cpp/vs version.